apps/web/public/static/locales/en/common.json

```diff
@@ -1957,7 +1957,7 @@
   "members_default_location": "Member's default location",
   "members_default_schedule_description": "We will use each members default availability schedule. They will be able to edit or change it.",
   "requires_at_least_one_schedule": "You are required to have at least one schedule",
-  "default_conferencing_bulk_description": "Update the locations for the selected event types",
+  "default_conferencing_bulk_description": "Update the schedules for the selected event types",
```

Comment:
This is incorrect. We have two places where this dialog is used, so we need two different descriptions. Revert this change and add description for the schedules in `locales/en`. Then, in `BulkEditDefaultForEventsModal`, add description props and pass the appropriate descriptions depending on the component. Use them accordingly

Also @SySagar next time complete things and open pr, as we don't review draft prs

Reply:
Just want to confirm. The other place where we are using keyword "location", is it Applist in installed apps sections when we are chagin default? I can't seem to find any otehr place where bulkModal is used

Reply:
Yess

Reply:
Currently we are using it in three places : availablitylist, applist and inside availability settings

Reply:
@anikdhabal please check.